app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+
+from app.cafe import Cafe
+from app.errors import (
+    NotWearingMaskError,
+    VaccineError
+)
+
+
+def go_to_cafe(friends: list, cafe: str) -> str:
```

Comment:
```suggestion
def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Your function takes Cafe instance